german/the-black-hack.md

```diff
@@ -216,7 +216,7 @@ Wenn Charaktere Waffen verwenden, die nicht bei ihrer Klasse aufgelistet stehen,
 
 Ein SL kann alle 15 Minuten in der realen Welt (ihr *haltet* das nach, oder?) einen W4 würfeln. Bei einem Ergebnis von 1-2 taucht innerhalb der nächsten **Minute** (Runde) ein zufällig ermitteltes Wesen oder eine Ablenkung auf.
 
-## Verhalten von Wesen
+## Verhalten von Kreaturen
```

Comment:
Für mich persönlich klingt "Kreatur" leicht diffamierend, "Wesen" finde ich neutraler, weswegen ich den Begriff instinktiv meide. Aber das ließe sich ändern, wenn du da einen Vorteil siehst. Es ist in der Vorlage offenbar als Catch-All-Begriff für beliebige Arten von NSCs gedacht, von Bauern und Kultisten bis hin zu Drachen, Orks und Geistern.